### DISCUSSION TITLE
QT support and Turtlesim

### DISCUSSION THREAD

**Discussion by AndreasKnoblach** [mvukov/rules_ros2#40] [Ideas]
Hi all,

I played around with Turtlesim tutorial using this bazel toolchain (which I pretty like).

I've a working example. I had to do one minor code change in the original files to work around an `AMENT_PREFIX_PATH not set` error. I guess there might be a proper solution for this, which I did not found.

I any case, if you'd be interested, I could setup a PR to add this to the examples and we could workout a pretty solution for the `AMENT_PREFIX_PATH not set` issue.

WDYT?

Andreas

**Comment by mvukov** [comment]
Hi Andreas, please open a PR and let's iterate for there on.

**Reply by AndreasKnoblach** [reply]
Hi Milan, thx for coming back to me.

I've created https://github.com/mvukov/rules_ros2/pull/43

WDYT?

**Reply by mvukov** [reply]
Thanks, I'll reply directly on the PR.
